src/main/java/org/broadinstitute/hellbender/tools/funcotator/TranscriptSelectionMode.java

```diff
@@ -381,6 +401,7 @@ public BestEffectGencodeFuncotationComparator( final Set<String> userRequestedTr
             byTranscriptName = new ComparatorByTranscriptName();
 
             chainedComparator = byUserTranscript
+                    .thenComparing(byManeStatus)
```

Comment:
Won't this now always prefer MANE transcripts, regardless of whether the prefer-MANE flag is set?

Are there any tests that run w/o the prefer-MANE flag, but with MANE transcripts in a scenario where a non-MANE transcript should be selected?